Resources/Prototypes/_RMC14/Entities/Clothing/Head/cmb.yml

```
@@ -0,0 +1,12 @@
+- type: entity
+  parent: ArmorHelmetM10
+  id: CMArmorHelmetM10CMB
```

```suggestion
  id: RMCArmorHelmetM10CMB
```
Use `RMC` instead of `CM` for new prototypes.